subprojects/diagnostics/src/integTest/groovy/org/gradle/api/tasks/diagnostics/OutgoingVariantsReportTaskIntegrationTest.groovy

```diff
@@ -248,6 +256,8 @@ Variant runtimeElements
 --------------------------------------------------
 Description = Elements of runtime for main.
 
+Capabilities
+    - [default capability]
```

Comment:
:o: I would prefer if this still prints the coordinates of the default capability. Since users currently are not familiar with the concept, I think it will help to understand. For example if you want to provide an alternative implementation that uses the default capability of another module.

For example:

```
Capabilities
    - org:myLib:1.0 (default capability)
```

Reply:
Done